### PR TITLE
Fixed listings/dashboard/rowElements/location.jsx eslint error

### PR DESCRIPTION
**Pull request by JuanVqz**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Fixed listings/dashboard/rowElements/location.jsx eslint error

## Related Tickets & Documents
#2470 
## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [x] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [ ] no documentation needed

### CODE REVIEW

**Comment by JuanVqz** [comment]
@jacobherrington thank you! How or where can I see that page?

**Comment by jacobherrington** [comment]
@JuanVqz This component gets rendered at `http://localhost:3000/listings/dashboard`. You'll need to make a listing locally for it to show up!

File: app/javascript/listings/dashboard/rowElements/location.jsx [hunk 1]
```diff
@@ -2,16 +2,11 @@ import PropTypes from 'prop-types';
...
 
 const Location = ({ location }) => {
-  return (
-    <span className="dashboard-listing-date">
-       ・ 
-      {location}
-    </span> 
-  )
-}
+  return <span className="dashboard-listing-date">{location}</span>;
```
Comment (nickytonline) [inline app/javascript/listings/dashboard/rowElements/location.jsx]:
```suggestion
	return <span className="dashboard-listing-date">・{location}</span>;
```

**Comment by nickytonline** [comment]
Not enough coffee today @jacobherrington. I missed the removal of the `・`. 🙃

**Pull request merged**